src/iidm/NodeBreakerVoltageLevelViews.cpp

```diff
@@ -285,13 +289,12 @@ NodeBreakerViewImpl::SwitchAdder NodeBreakerViewImpl::newSwitch() {
     return SwitchAdder(m_voltageLevel);
 }
 
-void NodeBreakerViewImpl::removeSwitch(const std::string& switchId) {
-    m_voltageLevel.removeSwitch(switchId);
+void NodeBreakerViewImpl::removeInternalConnections(unsigned long node1, unsigned long node2) {
+    m_voltageLevel.removeInternalConnections(node1, node2);
```

Comment:
Maybe the removeInternalConnections implementation should be moved here: you only need the graph?

Reply:
removeInternalConnections modifies the graph. Is the view allowed to do that ?